utils/installer/install.sh

```diff
@@ -89,6 +103,8 @@ installpynvim() {
     if [ -f "/etc/gentoo-release" ]; then
         echo "Installing using Portage"
         sudo emerge -avn dev-python/pynvim
+    elif [ "$(uname -s)" == "FreeBSD" ]; then
+        sudo pkg install py38-pynvim
```

Comment:
If I understand correctly, the pip3 symlink is not created when installing the package.
Would it be possible to make one in a directory listed in $PATH ? This way we could remove the || conditions below